packages/server-renderer/src/helpers/ssrRenderAttrs.ts

```diff
@@ -35,7 +35,7 @@ export function ssrRenderAttrs(
       continue
     }
     const value = props[key]
-    if (key === 'class') {
+    if (key === 'class' || key === 'className') {
```

Comment:
this is a breaking change

Reply:
Even though it appears to be the correct behavior on the client side?

Reply:
Here is a bit of a difference, maybe we can ignore it.
- [Playground](https://play.vuejs.org/#__SSR__eNp9UsFOwzAM/ZUol15Ge4BT6SYBmgQcBgJuhENpvZKRJlHilqFp/46TslGhbTfb7zl+8fOGX1mb9h3wnBe+ctIi84CdnQktW2scshvTWrZ0pmVJmoUk0BOhK6M9shLReTZlG6EZq1Tp/aJsIX9NHNTJJGkcgE7ehN4KXWTDAHqaEoTWqhKBMsaKOKQ/e5e6ngoeHxU8I6zIRkQ+4ehp7lI26cobTaLjXMEr6pcK3INFSboEzwdFASuVMl/3sYaug8muXn1A9XmgvvLrUBP80YEH14PgewxL1wAO8Px5AWuK92Br6k4R+wT4BN6oLmgcaNedrkn2iBfV3sXVS928+PkaQfvdp4LQwNxGvuBkRVjdsa//yT1PL2IfOUFb3Nl4wPVTLtWyn90CCSyyEP5zp/D4rUKQkvfDPRhlXE7ZZTyANF7DCGGxMIA097edb38AeH7gQA==)
- [Playground with this PR](https://deploy-preview-11722--vue-sfc-playground.netlify.app/#__SSR__eNp9UsFOwzAM/ZUol15Ge4BT6SYBmgQcBgJuhENpvZKRJlHilqFp/46TslGhbTfb7zl+8fOGX1mb9h3wnBe+ctIi84CdnQktW2scshvTWrZ0pmVJmoUk0BOhK6M9shLReTZlG6EZq1Tp/aJsIX9NHNTJJGkcgE7ehN4KXWTDAHqaEoTWqhKBMsaKOKQ/e5e6ngoeHxU8I6zIRkQ+4ehp7lI26cobTaLjXMEr6pcK3INFSboEzwdFASuVMl/3sYaug8muXn1A9XmgvvLrUBP80YEH14PgewxL1wAO8Px5AWuK92Br6k4R+wT4BN6oLmgcaNedrkn2iBfV3sXVS928+PkaQfvdp4LQwNxGvuBkRVjdsa//yT1PL2IfOUFb3Nl4wPVTLtWyn90CCSyyEP5zp/D4rUKQkvfDPRhlXE7ZZTyANF7DCGGxMIA097edb38AeH7gQA==)

Reply:
I'm sorry I didn't notice this situation. I found that although `className` is converted to `class` in csr, it's not processed correctly when it's not of string type. I'm not sure if this is another issue.

Reply:
To be consistent, `className` should not go through `ssrRenderClass` which normalizes non-string values into strings. For `className` it should coerce directly into strings with `String(value)`.